### DISCUSSION TITLE
What to gitignore and what not to (helm)

### DISCUSSION THREAD

**Discussion by blmhemu** [grafana/tanka#1113] [General]
Should the charts folder be checked in to git ? Or is the recommendation same as that of vendor ?

**Comment by zerok** [comment]
Hi 👋 

We check the chart folders into Git so that changes are easier to track. But if your goal is to have a really minimal repository without vendored items, then don't 🙂 

I don't think we have an official recommendation but _personally_ I'd check them in whenever possible.

**Reply by blmhemu** [reply]
Makes sense ! Thanks for the quick reply.
